activestorage/activestorage.gemspec

```diff
@@ -37,4 +37,5 @@ Gem::Specification.new do |s|
   s.add_dependency "activerecord",  version
 
   s.add_dependency "marcel", "~> 0.3.1"
+  s.add_dependency "rack", "~> 2.0", ">= 2.1.0"
```

Comment:
Rack is used in `FileServer`, which is used in `DiskController`.
I tend to think that It's better to specify the dependency explicitly even when `DiskController` is not used rather than get a message that `Rack::Files` is undefined when `rack` used happens to be < 2.1.0.